<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_app_view</class>
 <widget class="QMainWindow" name="main_app_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Report generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="page_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>881</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(131, 216, 255)
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ระบบลงทะเบียน</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="customer_reg_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ลงทะเบียนลูกค้าใหม่</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="case_register_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>รับงานใหม่</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="check_job_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ตรวจสอบงาน</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="barcode_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ปริ้นบาร์โค้ด</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lab_order_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>พิมพ์ใบส่งแลป</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bill_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ออกใบเสร็จ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="check_report_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>รายงานที่ส่งแล้ว</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="employee_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>จัดการข้อมูลบุคลากร</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="personal_info_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>แก้ไขข้อมูลส่วนตัว</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="update_prog_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>UPDATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sign_out_pushButton">
       <property name="font">
        <font>
         <family>TH Niramit AS</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sign Out</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/icons/log_out_white.png</normaloff>:/icons/icons/log_out_white.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>1201</width>
      <height>851</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1181</width>
       <height>831</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>10</number>
     </property>
     <widget class="QWidget" name="login_page">
      <widget class="QLineEdit" name="username_lineEdit">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>260</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>310</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="login_pushButton">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>340</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>240</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>290</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>210</y>
         <width>291</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
      <zorder>groupBox</zorder>
      <zorder>username_lineEdit</zorder>
      <zorder>password_lineEdit</zorder>
      <zorder>login_pushButton</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
     </widget>
     <widget class="QWidget" name="new_customer_page">
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>New customer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="new_job_page">
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>New job</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="check_job_page">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Check Job progress</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="employee_page">
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Employee</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="check_report_page">
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Check report</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bill_page">
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Bill page</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="barcode_page">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Barcode page</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="lab_report_page">
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Lab report</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="edit_personal_page">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Personal</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="update_page">
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Update page</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="top_frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>1201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="current_user_label">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionCytology_Report">
   <property name="text">
    <string>Cytology Report</string>
   </property>
  </action>
  <action name="actionNecropsy_Report">
   <property name="text">
    <string>Necropsy Report</string>
   </property>
  </action>
  <action name="actionSave_on_This_PC">
   <property name="text">
    <string>Save on This PC</string>
   </property>
  </action>
  <action name="actionSave_on_Server">
   <property name="text">
    <string>Save on Server</string>
   </property>
  </action>
  <action name="actionLoad_from_this_PC">
   <property name="text">
    <string>Load from this PC</string>
   </property>
  </action>
  <action name="actionLoad_from_Server">
   <property name="text">
    <string>Load from Server</string>
   </property>
  </action>
  <action name="actionConfirm_Report">
   <property name="text">
    <string>Confirm Report</string>
   </property>
  </action>
  <action name="actionApprove_Report">
   <property name="text">
    <string>Approve Report</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
